### PR TITLE
chore(common): add script to update library version in templates

### PR DESCRIPTION
**Pull request by eunjae-lee**
## What?

`yarn bump-version` now asks you if you want to update `@storyblok/field-plugin` version in all the templates in case of bumping the CLI version.

## Why?

JIRA: EXT-1950

After you release `@storyblok/field-plugin`, you normally want to release `@storyblok/field-plugin-cli`. When you do, you need to upgrade `@storyblok/field-plugin` to the latest version you just released. This PR updates `bump-version` script to let you upgrade the version automatically.

### CODE REVIEW

File: scripts/bump-version.mjs [hunk 1]
```diff
@@ -163,6 +163,19 @@ if (prerelease) {
   nextVersion = result.nextVersion
 }
 
+if (packageFolder === 'cli') {
```
Comment (demetriusfeijoo) [inline scripts/bump-version.mjs:166]: @eunjae-lee, could we have an `enum` for this checking here?

```
const PACKAGE_FOLDER_TYPES = {
    FIELD_PLUGIN: 'field-plugin',
    CLI: 'cli',
}

...

if (packageFolder === PACKAGE_FOLDER_TYPES.CLI) {}
```

and then we could also update the `PACKAGE_FOLDERS` for make use of it...
something like that:

```
const PACKAGE_FOLDERS = [
  {
    title: 'Library',
    value: PACKAGE_FOLDER_TYPES.FIELD_PLUGIN,
  },
  {
    title: 'CLI',
    value: PACKAGE_FOLDER_TYPES.CLI,
  },
]
```

What do you think?

Comment (eunjae-lee) [reply scripts/bump-version.mjs:171]: hey @demetriusfeijoo based on your feedback, I just converted this to typescript!

Comment (demetriusfeijoo) [reply scripts/bump-version.mjs:173]: Thanks @eunjae-lee. 

Nice job changing it.

I was checking it locally and now it seems the ESLint is not working anymore for this script:

image

Comment (eunjae-lee) [reply scripts/bump-version.mjs:175]: fixed! https://github.com/storyblok/field-plugin/pull/265/commits/421de3ffba8e48d1dd54dcc00188bee7edce6090

**Pull request merged**